src/main/java/org/opentripplanner/updater/stoptime/TimetableSnapshotSource.java

```diff
@@ -815,7 +875,12 @@ private boolean cancelPreviouslyAddedTrip(FeedScopedId tripId, final ServiceDate
       final Timetable timetable = buffer.resolve(pattern, serviceDate);
       final int tripIndex = timetable.getTripIndex(tripId);
       if (tripIndex == -1) {
-        LOG.warn("Could not cancel previously added trip {}", tripId);
+        warn(
```

Comment:
Should we have `public static void warn(FeedScopedId tripId, String message, Object... params)` for this? Also now the trip id is duplicated, maybe replace with the serviceDate

Reply:
I've added that message and used the service date instead.